tests/unittests/tests-base64/tests-base64.c

```diff
@@ -414,7 +414,11 @@ static void test_base64_11_urlsafe_encode_int(void)
     unsigned char expected_encoding[] = "-RAAAA";
 
     size_t base64_out_size = 0;
-    char base64_out[sizeof(expected_encoding)];
+
+    /* Up to three = signs are suppressed in urlsafe encoding at the end, but
+     * still written in the implementation. Just allocate 3 bytes more as worst
+     * case */
+    char base64_out[sizeof(expected_encoding) + 3];
```

Comment:
```suggestion
    /* Up to two "=" signs are suppressed in urlsafe encoding at the end, but
     * still written in the implementation. Just allocate 2 bytes more as worst
     * case */
    char base64_out[sizeof(expected_encoding) + 2];
```

I just realized that encoding one byte takes two data chars and two "=" padding chars, so a padding of two "=" is already the worst case.

Reply:
squashed right in